### ISSUE TITLE
Add support for additional TFR methods

### ISSUE THREAD

**Issue by tsbinns** [mne-tools/mne-connectivity#126]
Currently, `cwt_morlet` is the only TFR method supported, and I see there is a note in `epochs.py`: `# reminder to add alternative TFR methods`.

Would it be possible to add support for e.g. the [multitaper TFR method available in the `mne-base` package](https://mne.tools/dev/generated/mne.time_frequency.tfr_array_multitaper.html)?

Best,
Thomas!

**Comment by adam2392** [comment]
Yes that is a good idea. A PR is welcome! :) Lmk if you need guidance

**Comment by tsbinns** [comment]
Gonna close this as completed now, since with #232 `EpochsTFR` objects are supported by the spectral connectivity functions, allowing the multitaper TFR method to be used.
